src/app/router/Options/Options.tsx

```diff
@@ -56,28 +58,43 @@ function Options() {
               <Route
                 path="new"
                 element={
-                  <Container>
+                  <Container maxWidth="xl">
                     <Outlet />
                   </Container>
                 }
               >
                 <Route
                   index
                   element={
-                    <ChooseConnector
-                      title={i18n.t(
-                        "translation:choose_connector.title.options"
+                    <ChooseConnectorPath
+                      title={i18n.t("translation:choose_path.title")}
+                      description={i18n.t(
+                        "translation:choose_path.description"
                       )}
                     />
                   }
                 />
-                {connectorRoutes.map((connectorRoute) => (
+                <Route path="create-wallet" element={<NewWallet />} />
```

Comment:
Wondering if `create-wallet` (also in combination with `NewWallet`) is confusing.  
This will lead to "Alby: new wallet / login to exisiting wallet", right?  

Would something like this reflect this better?:
```ts
<Route path="albvy-wallet" element={<AlbyWallet />} />
```

Reply:
This will have to be undone immediately after we split the create and login account right? I am not sure if it is worth it, if I work on that task immediately after this one.

Reply:
Right, good point!